### PR TITLE
Setup script for RDS and update Readme with instruction

### PR DESCRIPTION
**Pull request by yonassrobi**
Terraform code for RDS setup and update readme file with AWS setup and teardown instructions.

### CODE REVIEW

**Review by caoabunga — APPROVED** [review]

File: setup/main.tf [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  egress {
+    from_port   = 3306
+    to_port     = 3306
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "education_rds"
```
Comment (TheDanMiller) [inline setup/main.tf:95]: @yonassrobi what is an education_rds? Should this be tagged differently

File: setup/main.tf [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+}
+
+resource "aws_security_group" "acme-rds-sg" {
+  name       = "acme-${var.environment}-rds-sg"
+  vpc_id = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 3306
+    to_port     = 3306
+    protocol    = "tcp"
```
Comment (TheDanMiller) [inline setup/main.tf:83]: @yonassrobi should we lock this down to only the container that is running the API? With security as a forefront item this seems wide open

**Review by TheDanMiller — CHANGES_REQUESTED** [review]

File: setup/main.tf (hunk 2 above)
Comment (yonassrobi) [reply setup/main.tf:83]: Agreed. This has been updated.

File: setup/main.tf (hunk 1 above)
Comment (yonassrobi) [reply setup/main.tf]: This was a type. It has been updated to use the correct tags.

**Review by TheDanMiller — APPROVED** [review]

**Pull request merged**